packages/blocks/src/api/raw-handling/test/list-reducer.js

```diff
@@ -41,12 +41,6 @@ describe( 'listReducer', () => {
 		expect( deepFilterHTML( input, [ listReducer ] ) ).toEqual( output );
 	} );
 
-	it( 'should remove empty list wrappers', () => {
-		const input = '<ul><li>\n<ul><li>test</li></ul>\n</li></ul>';
-		const output = '<ul><li>test</li></ul>';
-		expect( deepFilterHTML( input, [ listReducer ] ) ).toEqual( output );
-	} );
```

Comment:
The important tests are actually the integration tests, and curiously this doesn't make any integration tests fail, so it seems fine to leave this alone.